src/pages/edge/edge-network/datastream-config-override-using-tags.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Datastream config override for Edge events using Tags.
+description: This guide provides instructions on overriding datastream ID and datastream configuration for Edge Network Requests that are not initiated by the sendEvent API, all accomplished through Tags Rules.
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Overriding Datastream ID and Configuration for Edge Network Requests Using Tags Rules
+
+<InlineAlert variant="info" slots="text"/>
+
+Starting with version **2.4.0** of the `Edge` Android extension and version **4.3.0** of the `AEPEdge` iOS extension, the `sendEvent` API now supports optional Datastream overrides. For detailed information, please refer to the [sendEvent API reference](./api-reference.md#sendEvent).
+
+## Using Datacollection Tags Rules for datastream configuration overrides in Edge based extensions
+
+Suppose you are working with an Edge based extension like **Streaming Media for Edge** and you need to override `datastream ID` and `datastream configuration` for Media requests passing through the Edge Network. Follow these steps to achieve that:
+
+1. Navigate to the **DataCollection UI** and go to **Tags** select your mobile property.
+
+2. Add a new rule for your mobile property:
+
+    a. Provide a descriptive name for your Rule.
+
+    b. Under the **If** section in the **Events** category, add a new Event.
+
+    c. In the Event configuration view, set the **Extension** to "Adobe Experience Platform Edge Network" and the **Event Type** to "AEP Request Event." Then, click the plus button to "Add XDM Event Type." For all media requests, set the condition to contain "media" since all media events have types prefixed with "media."
+
+    ![Sample Event configuration for Rule](./assets/configOverrides/rule-eventConfiguration-edge-typeMedia.png)
+
+    d. Save the event configuration and return to the Rule Editor UI.
+
+    e. In the **Then** section under **Actions** add a new Action.
+
+    f. In the Action configuration view, set the **Extension** to "Mobile Core" and the **Action Type** to "Attach Data."
+
+    g. Now, add the JSON payload for your `datastream ID override` and `datastream config override` in the following format:
```

Comment:
Similar to the API docs, can we split this into two distinct examples - one for datastreamIdOverride and one for datastreamConfigOverride